cg/services/delivery_message/messages/order_message.py

```diff
@@ -0,0 +1,22 @@
+from cg.constants import DataDelivery
+from cg.services.delivery_message.messages.delivery_message import DeliveryMessage
+from cg.services.delivery_message.messages.utils import get_caesar_delivery_path
+from cg.store.models import Case
+
+
+class OrderDeliveryMessage(DeliveryMessage):
+
+    FILE_TYPES_BY_DATA_DELIVERY_MAP: dict[DataDelivery, str] = {
+        DataDelivery.FASTQ_ANALYSIS: "fastq and analysis",
+        DataDelivery.FASTQ: DataDelivery.FASTQ,
+        DataDelivery.ANALYSIS: DataDelivery.ANALYSIS,
+    }
+
+    def create_message(self, cases: list[Case]) -> str:
+        delivery_path: str = get_caesar_delivery_path(cases[0])
+        file_types: str = self.FILE_TYPES_BY_DATA_DELIVERY_MAP[cases[0].data_delivery]
```

Comment:
```suggestion
FILE_TYPES_BY_DATA_DELIVERY_MAP: dict[DataDelivery, str] = {
        DataDelivery.FASTQ_ANALYSIS: "fastq and analysis"
    }

(...)
        data_delivery = cases[0].data_delivery
        file_types: str = self.FILE_TYPES_BY_DATA_DELIVERY_MAP.get(data_delivery, data_delivery)
```

Could do this but then there is not much sense in having a map? These are the only possible delivery types for taxprofiler at the moment (the only workflow to have this kind of message), but having a dict like this could be useful to add more data delivery types in the future?

Reply:
I prefer it as you have it in the PR!

Reply:
Nice, thanks!